### ISSUE TITLE
Will Genesis support USD prims/assets?

### ISSUE THREAD

**Issue by mouhyemen** [Genesis-Embodied-AI/Genesis#94]
Hi,

Great work. Will Genesis support USD assets in the future?

**Comment by YilingQiao** [comment]
Yeah, I think we will support it in the near future when we start using a large amount of USD assets. Until then, we also welcome PRs for that!

**Comment by kangxiatao** [comment]
非常期待。
